templates/block-js.mustache

```diff
@@ -0,0 +1,36 @@
+( function( wp ) {
+	var el = wp.element.createElement;
+	var __ = wp.i18n.__;
+
+	// Visit https://wordpress.org/gutenberg/handbook/block-api/ to learn about Block API
```

Comment:
Should this be linked to in the `wp scaffold block` command description?

Reply:
We should add it there, too.